### ISSUE TITLE
Authorize.net - Network Change Notification

### ISSUE THREAD

**Issue by kdjstudios** [kdjstudios/SABillingGitlab#1834]
In GitLab by @kdjstudios on Jan 14, 2020, 11:18

**Submitted by:** Tim

**Helpdesk:** http://www.servicedesk.answernet.com/profiles/ticket/2020-01-14-96579

**Server:** Both

**Client/Site:** NA 

**Account:** NA

**Issue:** 

[original_message__3_.html](/uploads/6e3262c613be6453fce5342e93ff10f1/original_message__3_.html)

http://app.payment.authorize.net/e/es.aspx?s=986383348&e=1761522&elqTrackId=b4d9a2a0f1424fe28f720ff9d8ae4d54&elq=a547e89316b149a4b6b09518ee61b228&elqaid=963&elqat=1

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Jan 14, 2020, 11:19

@smasih @jredfield 

Please confirm if SA Billing needs to update anything to continue working with Authroize.net. I couldn't get it to copy paste nicely, so I just linked to the full email above.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Mar 3, 2020, 10:51

@amishra @smasih @jredfield @ranand @rranjan 

May we please have an update?

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Mar 11, 2020, 13:49

@amishra @smasih @jredfield @ranand @rranjan

May we please have an update?

It stated in their email "Production is targeted for February and March 2020". So we are already behind on this and issues may start occurring now since no one has looked into this yet.

**Comment by kdjstudios** [comment]
In GitLab by @ranand on Mar 12, 2020, 04:27

@jredfield @kdjstudios @tim.traylor 

We looked into it, there is no code level change in it. There is a change in the API endpoints URLs on Akamai and the API endpoints not on Akamai will not be changing. We are not directly using their API endpoints, we are accessing them through authorize.net ruby-SDK/Gem which is not on Akamai. So far they have not released any new version of the gem regarding this.

There is also a trust certification upgrade on the server-side. This needs to be done by the System administrator, we don't have expertise in it in our SAB team, so this needs to be done by Answernet server/system administrator.

We checked the payments are working. I would suggest after the upgrade of mongo and rails, we should upgrade the authorize.net gem as well. 

Thanks
